docs/management/connectors/action-types/servicenow-itom.asciidoc

```diff
@@ -79,62 +83,6 @@ JWT Key ID:: The key ID assigned to the JWT verifier map of your OAuth applicati
 Private Key:: The RSA private key generated during setup.
 Private Key Password:: The password for the RSA private key generated during setup, if set.
 
-[float]
-[[preconfigured-servicenow-itom-configuration]]
-=== Create preconfigured connectors
-
-If you are running {kib} on-prem, you can define connectors by
-adding `xpack.actions.preconfigured` settings to your `kibana.yml` file.
-For example:
-
-Connector using Basic Authentication
-[source,text]
---
-xpack.actions.preconfigured:
-  my-servicenow-itom:
-    name: preconfigured-servicenow-connector-type
-    actionTypeId: .servicenow-itom
-    config:
-      apiUrl: https://example.service-now.com/
-    secrets:
-      username: testuser
-      password: passwordkeystorevalue
---
-
-Connector using OAuth
-[source,text]
---
- my-servicenow:
-   name: preconfigured-oauth-servicenow-connector-type
-   actionTypeId: .servicenow-itom
-   config:
-     apiUrl: https://example.service-now.com/
-     usesTableApi: false
```

Comment:
> usesTableApi: false

Is `usesTableApi` a valid setting for ServiceNow ITOM? When I try using it in the API, I get "error validating action type config: [usesTableApi]: definition for this key is missing".

Reply:
Good catch! It looks like `usesTableApi` is only valid for ServiceNow SIR and ServiceNow ITSM